rust_src/src/indent.rs

```diff
@@ -103,8 +103,10 @@ pub fn move_to_column(column: EmacsInt, force: bool) -> EmacsInt {
 
                 // Delete the tab and indent to COL
                 del_range(buffer.pt, buffer.pt + 1);
+                let goal_pt = buffer.pt;
```

Comment:
Why add the temporary variables?

Reply:
Because `Findent_to` changes the value of `buffer.pt{,_byte}`

Reply:
Ah, ok.